sync_diff_inspector/source/source.go

```diff
@@ -130,19 +129,6 @@ func NewSources(ctx context.Context, cfg *config.Config) (downstream Source, ups
 			Collation:           tableConfig.Collation,
 			ChunkSize:           tableConfig.ChunkSize,
 		})
-
```

Comment:
we do have another requirement that ignore insensitive table name with sync-diff.
so we add the default rule to make sure the same name between upstream and downstream can be handle correctly.

@leoppro WDYT?